webapp/components/compat-2021.js

```diff
@@ -44,14 +113,73 @@ class Compat2021 extends PolymerElement {
   static get properties() {
     return {
       stable: Boolean,
+      feature: String,
     };
   }
 
+  static get observers() {
+    return [
+      'updateUrlParams(stable, feature)',
+    ];
+  }
+
   ready() {
     super.ready();
 
     const params = (new URL(document.location)).searchParams;
     this.stable = params.get('stable') !== null;
+    this.feature = params.get('feature');
+
+    // The default behavior of the page (when loaded with no params) is to not
```

Comment:
This was added earlier and just moved now, but I don't understand the comment. Setting `value` has the same effect regardless isn't it? Or is this in reference to some work that isn't being done which the reader would suspect is needed but actually isn't?